app/src/main/java/ru/flicksbox/movie/data/MovieService.kt

```diff
@@ -0,0 +1,12 @@
+package ru.flicksbox.movie.data
+
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+import ru.flicksbox.network.BaseDTO
+
+interface MovieService {
+
+    @GET("api/v1/movies/top")
+    suspend fun getTopMovies(@Query("count") count: Int, @Query("from") from: Int): Response<BaseDTO<MoviesDto>>
```

Comment:
`BaseDTO`
`MoviesDto`

In first case you've written `DTO`, in second `Dto`. Please, use only one naming approach